pkg/test/resourcefixture/testdata/basic/privilegedaccessmanager/v1alpha1/privilegedaccessmanagerentitlement/_generated_object_privilegedaccessmanagerentitlement.golden.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: privilegedaccessmanager.cnrm.cloud.google.com/v1alpha1
```

Comment:
Yes. `spec.approvalWorkflow` and `spec.additionalNotificationTargets` are optional. Any reason you asked this question?

Reply:
Yes, we want to distinguish basic and full so that it is more straightforward to use as reference. You can see more detailed guide here https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/docs/develop-resources/guides/1-add-mockgcp-tests.md#a-basic-test-suite